### DISCUSSION TITLE
Country-based element selection for `ElementFilterExpression`

### DISCUSSION THREAD

**Discussion by burrscurr** [streetcomplete/StreetComplete#5670] [Ideas]
When looking into how to implement #5667 I thought that it would be cool to select (or exclude) elements based on the country they are in, for example like this: `nodes with traffic_sign=city_limit in DE` or `nodes with traffic_sign=city_limit not in PL`. Would an extension like this be helpful? I think this could make it easier to adjust quests to country-specific conditions besides the more coarse option to enable/disable a quest in its entirety.

**Comment by HolgerJeromin** [comment]
Do you have something like that in mind:

https://github.com/streetcomplete/StreetComplete/blob/918050acfb5390db4b7f1251e96067bb6e936d17/app/src/main/java/de/westnordost/streetcomplete/quests/accepts_cash/AddAcceptsCash.kt#L57-L62

**Reply by matkoniecz** [reply]
Note "adjust quests to country-specific conditions besides the more coarse option to enable/disable a quest in its entirety."

**Comment by mnalis** [comment]
- I thought that having such a feature in element selection parser would be a good idea too, but see https://github.com/streetcomplete/StreetComplete/issues/4528

**Comment by westnordost** [comment]
I'd rather not extend the element filter syntax like that. I don't see a reason why one wouldn't highlight the city limit sign in any country. If it is irrelevant for the speed limit in some countries, then the hightlighted element can just be ignored by users.

**Reply by burrscurr** [reply]
If there are no objections to highlighting all city limit signs I'm happy to implement it this way.

Edit: #5675
